roles/snmp/tasks/main.yml

```
@@ -0,0 +1,34 @@
+- block:
+
+  - name: 'Install snmp'
```

the name should be the same as a `dnf install ...`, without the dnf. so in this case:
```suggestion
  - name: 'Install net-snmp net-snmp-libs net-snmp-utils'
```